### PR TITLE
[gmock] Rename struct Result to avoid shadowing

### PR DESCRIPTION
**Pull request by TurboGawron**
Struct Result shadows global definition of class Result in gmock-spec-builders_test.cc inside ExpectCallTest.NonMoveableType test

### CODE REVIEW

**Review by ankit-gautam23 — CHANGES_REQUESTED** [review]
Changing just the case won't help much. We can think of some better name.

**Comment by TurboGawron** [comment]
@ankit-gautam23  I agree, so maybe let's change that struct name to something to one of this: NonMoveableType, NonMoveableStruct, NonMoveableData, NonMoveableTestData or just NonMoveable. Or perhaps you have diffrent proposition? 
Sorry I dind't respond yesterday I was short on time after my work

**Comment by ankit-gautam23** [comment]
@TurboGawron Its truly okay. I will update this and raise the PR. Will add you as a reviewer. Thanks

File: googlemock/test/gmock-spec-builders_test.cc [hunk 1]
```diff
@@ -811,32 +811,32 @@ TEST(ExpectCallTest, InfersCardinality1WhenThereIsWillRepeatedly) {
 // from a prvalue returned from a function.
 TEST(ExpectCallTest, NonMoveableType) {
   // Define a non-moveable result type.
-  struct Result {
-    explicit Result(int x_in) : x(x_in) {}
-    Result(Result&&) = delete;
+  struct result {
```
[the remark this answers is not in the document]
Comment (TurboGawron) [reply googlemock/test/gmock-spec-builders_test.cc:814]: Ok

**Pull request closed without merging**